### ISSUE TITLE
Invalide file path for venv\Scripts\python.exe

### ISSUE THREAD

**Issue by vl2376** [spyder-ide/spyder#14754]
### What steps will reproduce the problem?

Tough question. Trying to specify my python executable (within a venv). Using preferences > python interpreter and windows file explorer opened by spyder to click on my `python.exe` file within `venv/Scripts`. I get `invalid file path error` prompted on a window. Also it's working for other `venv`, though I have just created this one with an updated version of `virtualenv`

## Versions

* Spyder version: 4.2.1 
* Python version: 3.6.8
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.16.1 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  22.0.1 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.7.4 (OK)
pylint >=1.0                    :  2.6.0 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.0             :  0.3.0 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.1 (OK)
qtconsole >=5.0.1               :  5.0.2 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.4 (OK)
setuptools >=39.0.0             :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                  :  3.4.3 (OK)
spyder_kernels >=1.10.1;<1.11.0 :  1.10.1 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3               :  1.0.2 (OK)
zmq >=17                        :  20.0.0 (OK)

# Optional:
cython >=0.21                   :  0.29.14 (OK)
matplotlib >=2.0.0              :  None (NOK)
numpy >=1.7                     :  1.19.0 (NOK)
pandas >=1.1.1                  :  0.25.3 (NOK)
scipy >=0.17.0                  :  None (NOK)
sympy >=0.7.3                   :  None (NOK)
```

**Comment by steff456** [comment]
Hi @vl2376,

Did you check that the route to the `venv` in your system is correct? If so, can you please add some screenshots in this issue to see if we are able to reproduce it?

**Comment by vl2376** [comment]
Thank you for helping me @steff456 

- The path is correct
- Here is a screenshot

![image](https://user-images.githubusercontent.com/31713734/107980228-2cbe3900-6fc0-11eb-8c17-870557664e42.png)

- A few other elements:
-- I have tried with \ or // ; still wont work
-- Everything was working fine Friday. Now I am facing the issue for all of my envs using virtualenv. Looking at it more thoroughly, it seems that it comes from going to spyder 4.1.5 to 4.2.1 ; I have downgraded back, but now I am facing an issue with spyder-kernels...

**Comment by steff456** [comment]
If you used the dropdown, does that entry appears in the automatic Spyder venv discovery?

**Comment by vl2376** [comment]
What do you mean ? What's the venv discovery ? Also pyflakes doesn't seem to work on 4.2.1

**Comment by steff456** [comment]
![image](https://user-images.githubusercontent.com/20992645/107983673-45185e80-6f94-11eb-81d6-ea1dba82fa9b.png)

In the latest release, we introduced a virtual and conda envs automatic discovery. You can check that in the dropdown in Preferences > Python Interpreter > Use the following Python Interpreter. Please let me know if you have the same entry that's not working or if the ones that appear in that widget work.

**Comment by vl2376** [comment]
Well nothing but this path appears in that dropdown folder, strangely...

**Comment by vl2376** [comment]
Also I have this message on top of my console:

`ImportError: numpy.core.multiarray failed to import`

**Comment by steff456** [comment]
Can you reset your configurations of spyder with `spyder --reset` and upload a screenshot of what you see in the dropdown? Please note to do this with Spyder 4.2.1

**Comment by ccordoba12** [comment]
> ImportError: numpy.core.multiarray failed to import

This looks like an installation issue due to mixing pip and conda, so I'd say your environment is broken.

**Comment by JoseCuervo92008** [comment]
Hello, 

In my case I got the "No spyder-kernels found" in Ipython console as well, even though my python path was setup correctly and the spyder_kernels folder was there. I saw that version was 1.0 with notepad. 

The error message itself was a little wrong, updating my spyder-kernels from 1.0 to 2.0 solved it. 

->Open cmd (in windows)
py
import pip
pip.main(['install','--upgrade','spyder-kernels']) 

Make sure it's the right python path!
